modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js

```diff
@@ -380,6 +381,11 @@ const Select = ({
 		setSelectedLabel('');
 	}, [currentValue, options, value]);
 
+	useEffect(() => {
+		const [selectedValue] = currentValue;
+		onChange({target: {value: [selectedValue]}});
+	}, []);
```

Comment:
As Carol has said, to avoid SF complain due to missing dependencies, add this line `// eslint-disable-next-line react-hooks/exhaustive-deps`
```suggestion
	useEffect(() => {
		const [selectedValue] = currentValue;
		onChange({target: {value: [selectedValue]}});
		// eslint-disable-next-line react-hooks/exhaustive-deps
	}, []);
```

Also, run `yarn test Select` to check the test failures related to the fix.
Please contact me if there's something I can help you.